### PR TITLE
some care for pep8 E262 (comments)

### PR DESCRIPTION
**Pull request by fchapoton**
some care (partial) for pep8 code E262

E262 inline comment should start with '# '

### :memo: Checklist

- [x] The title is concise and informative.
- [x] The description explains in detail what this PR is about.

### CODE REVIEW

File: src/sage/coding/guruswami_sudan/interpolation.py [hunk 1]
```diff
@@ -284,10 +284,10 @@ def gs_interpolation_linalg(points, tau, parameters, wy):
     # Pick a nonzero element from the right kernel
     sol = Ker.basis()[0]
     # Construct the Q polynomial
-    PF = M.base_ring()['x', 'y'] #make that ring a ring in <x>
+    PF = M.base_ring()['x', 'y']  # make that ring a ring in <x>
     x, y = PF.gens()
-    Q = sum([x**monomials[i][0] * y**monomials[i][1] * sol[i] for i in range(0, len(monomials))])
-    return Q
+    return sum([x**monomials[i][0] * y**monomials[i][1] * sol[i]
+                for i in range(len(monomials))])
```
Comment (dcoudert) [inline src/sage/coding/guruswami_sudan/interpolation.py]:
```diff
-   return sum([x**monomials[i][0] * y**monomials[i][1] * sol[i]
-                for i in range(len(monomials))])
+   return sum([x**m[0] * y**m[1] * sol[i]
+                for i, m in enumerate(monomials)])
```

**Review by dcoudert — COMMENTED** [review]
a small remark. Otherwise, LGTM.

File: src/sage/coding/guruswami_sudan/interpolation.py (hunk 1 above)
Comment (fchapoton) [reply src/sage/coding/guruswami_sudan/interpolation.py]: voilà, merci

**Review by dcoudert — APPROVED** [review]
LGTM.

**Pull request merged**